Pear/SceneDelegate.swift

```diff
@@ -92,3 +94,47 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
     }
 
 }
+
+extension SceneDelegate: UNUserNotificationCenterDelegate {
+
+    private func setupLocalNotifications() {
+        let center = UNUserNotificationCenter.current()
+        center.requestAuthorization(options: [.alert, .sound]) { (granted, error) in
+            print("granted: \(granted)")
+        }
+        center.delegate = self
+        // get rid of previously scheduled notifications
+        center.removeAllDeliveredNotifications()
+        center.removeAllPendingNotificationRequests()
+        createNotifications(center: center, day: 2, hour: 8)
+        createNotifications(center: center, day: 4, hour: 14)
+        createNotifications(center: center, day: 6, hour: 12)
```

Comment:
You shouldn't create notifications if the user denied permission. It's bad practice and we don't like warnings in the console.